sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { RestrictionType } from "@sims/sims-db";
+
+/**
+ * Base DTO for application restriction bypass.
+ */
+export class ApplicationRestrictionBypassBaseAPIOutDTO {
+  id: number;
+  restrictionType: RestrictionType;
+  restrictionCode: string;
+}
```

Comment:
Unless there is a requirement to have a base DTO to abstract multiple objects, I wouldn't recommend creating one.

Reply:
Updated.